torch/csrc/jit/tensorexpr/expr.h

```diff
@@ -120,26 +100,68 @@ class TORCH_API ExprHandle {
   Expr* base_expr_node_ = nullptr;
 };
 
+// The underlying representation node to a Var.
+// Currently, each Var object represents a unique variable, even though the
+// names might be the same. We should consider add a unique_name as well.
+class Var : public ExprNode<Var> {
```

Comment:
I think `Var` is a stmt concept, perhaps move to stmt.h

Reply:
No, `Var` can be a node in `Expr` tree (a leaf node).